### DISCUSSION TITLE
We're still processing your application.

### DISCUSSION THREAD

**Discussion by AtanasijeDjumic** [community/community#87419] [GitHub Education]
### Hi everyone,

Hi. I'm trying to verify my account for student developer packs. I connected my student email on Wednesday, and its Monday now, so its been more than 72 hours. It still says "We're still processing your application. Once this is complete you will have access to the student developer pack offers." I heard that it can take more than 72 hours during peak times of the year, but I'm kind of low on time, so I would like to know if there's something i can do to solve my problem faster.

**Comment by anuragjain-git** [comment]
It usually take 72hr but during peek time like start of a semester it may take longer. You have to wait you can check its current status [here](https://education.github.com/discount_requests/application)

**Reply by AtanasijeDjumic** [reply]
I was hoping for a way to speed the process up but I guess I'll just have to wait

**Reply by anuragjain-git** [reply]
you can reach out to them through [link](https://support.github.com/contact)
